### PR TITLE
Regenerate Library 02/01/2017

### PR DESCRIPTION
**Pull request by efossier**
Upgrade guide: https://github.com/twilio/twilio-php/blob/caf1a9d6172cf6727823de6608e24257e2af6419/UPGRADE.md
Changes: https://github.com/twilio/twilio-php/blob/caf1a9d6172cf6727823de6608e24257e2af6419/CHANGES.md

Other misc changes:
renamed `Deserialize::is8601DateTime` to `Deserialize::dateTime` since that function can handle many different formats of dates

### CODE REVIEW

File: Twilio/Rest/Api/V2010/Account/MessageOptions.php [hunk 1]
```diff
@@ -39,14 +39,6 @@ public static function create($from = Values::NONE, $messagingServiceSid = Value
     public static function read($to = Values::NONE, $from = Values::NONE, $dateSentBefore = Values::NONE, $dateSent = Values::NONE, $dateSentAfter = Values::NONE) {
         return new ReadMessageOptions($to, $from, $dateSentBefore, $dateSent, $dateSentAfter);
     }
-
-    /**
-     * @param string $body The body
-     * @return UpdateMessageOptions Options builder
-     */
-    public static function update($body = Values::NONE) {
```
Comment (jingming) [inline Twilio/Rest/Api/V2010/Account/MessageOptions.php]: How come this was deleted?

File: AUTHORS.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ A huge thanks to all of our contributors:
 - D. Keith Casey, Jr.
```
Comment (jingming) [inline AUTHORS.md]: dupe

**Comment by jingming** [comment]
nit: the commits are logged from `efossier@twilio` which is not an authorized user of this repo.

File: Twilio/Rest/Api/V2010/Account/MessageOptions.php (hunk 1 above)
Comment (efossier) [inline Twilio/Rest/Api/V2010/Account/MessageOptions.php]: Because updating a message body was moved from a optional parameter to a required one i think.

**Pull request merged**